vispy/scene/visuals.py

```diff
@@ -103,7 +104,9 @@ def create_visual_node(subclass):
 
     # Decide on new class name
     clsname = subclass.__name__
-    assert clsname.endswith('Visual')
+    if not (clsname.endswith('Visual') and issubclass(subclass, Frozen)):
+        raise RuntimeError('Class "%s" must end with Visual, and must '
```

Comment:
It was originally because I had each `Visual` subclass `Frozen` individually, then figured out making `BaseVisual` subclass `Frozen` was simpler. I left this in mostly as a sanity check. It should never really be violated.